skbio/stats/distance/_base.py

```diff
@@ -487,6 +487,34 @@ def to_data_frame(self):
         """
         return pd.DataFrame(data=self.data, index=self.ids, columns=self.ids)
 
+    @experimental(as_of="0.5.0-dev")
+    def to_series(self):
+        """Create a ``pandas.Series`` from this ``DissimilarityMatrix``.
+
+        Returns
+        -------
+        pd.Series
+            ``pd.Series`` with IDs on index.
+
+        Examples
+        --------
+        >>> from skbio import DistanceMatrix
```

Comment:
Can you expand this example to use a 4x4 distance matrix? With a 3x3, the iteration order isn't clear for the pairs of IDs put on the index (row-major order with respect to the upper matrix triangle).

```python
>>> from skbio import DistanceMatrix
>>> dm = DistanceMatrix([[0, 1, 2, 3],
>>>                      [1, 0, 4, 5],
>>>                      [2, 4, 0, 6],
>>>                      [3, 5, 6, 0]], ids=['a', 'b', 'c', 'd'])
>>> dm.to_series()
a  b    1.0
   c    2.0
   d    3.0
b  c    4.0
   d    5.0
c  d    6.0
dtype: float64
```